values/binary_test.go

```diff
@@ -38,6 +39,10 @@ func TestBinaryOperator(t *testing.T) {
 		// float + float
 		{lhs: 4.5, op: "+", rhs: 8.2, want: 12.7},
 		{lhs: 4.5, op: "+", rhs: floatNullValue, want: floatNullValue},
+		{lhs: 1.0, op: "/", rhs: 0.0, want: math.Inf(1)},
```

Comment:
How does the CSV decoder handle infinite float values? I suspect it's likely fine since I think it uses the Go standard library to both output and parse the values, but I'd like us to add a test for this and modify the spec to explicitly list this behavior if it isn't already.